### PR TITLE
contributing: update links in badges

### PR DESCRIPTION
**Pull request by rst0git**
This pull request updates the links of badges in `CONTRIBUTING.md`.

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -1,7 +1,8 @@
-[![master](https://travis-ci.org/checkpoint-restore/criu.svg?branch=master)](https://travis-ci.org/checkpoint-restore/criu)
-[![development](https://travis-ci.org/checkpoint-restore/criu.svg?branch=criu-dev)](https://travis-ci.org/checkpoint-restore/criu)
-[![Codacy Badge](https://api.codacy.com/project/badge/Grade/55251ec7db28421da4481fc7c1cb0cee)](https://www.codacy.com/app/xemul/criu?utm_source=github.com&amp;utm_medium=referral&amp;utm_content=xemul/criu&amp;utm_campaign=Badge_Grade)
-<p align="center"><img src="https://criu.org/w/images/1/1c/CRIU.svg" width="128px"/></p>
+![X86_64 GCC Test](https://github.com/checkpoint-restore/criu/workflows/X86_64%20GCC%20Test/badge.svg)
+![Podman Test](https://github.com/checkpoint-restore/criu/workflows/Podman%20Test/badge.svg)
```
Comment (avagin) [inline CONTRIBUTING.md:2]: Where is Docker Test? ;)

Comment (rst0git) [reply CONTRIBUTING.md]: Thanks for asking ;)
I copied the badges from the readme file, but apparently the Docker Test badge was missing there. I have updated the the badges in both files now.

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -1,7 +1,9 @@
-[![master](https://travis-ci.org/checkpoint-restore/criu.svg?branch=master)](https://travis-ci.org/checkpoint-restore/criu)
-[![development](https://travis-ci.org/checkpoint-restore/criu.svg?branch=criu-dev)](https://travis-ci.org/checkpoint-restore/criu)
-[![Codacy Badge](https://api.codacy.com/project/badge/Grade/55251ec7db28421da4481fc7c1cb0cee)](https://www.codacy.com/app/xemul/criu?utm_source=github.com&amp;utm_medium=referral&amp;utm_content=xemul/criu&amp;utm_campaign=Badge_Grade)
-<p align="center"><img src="https://criu.org/w/images/1/1c/CRIU.svg" width="128px"/></p>
+![X86_64 GCC Test](https://github.com/checkpoint-restore/criu/workflows/X86_64%20GCC%20Test/badge.svg)
```
Comment (avagin) [inline CONTRIBUTING.md:1]: nit: I would remove these badges from CONTRIBUTING.md. We have them in README.md. But it is up to you.

File: CONTRIBUTING.md (hunk 1 above)
Comment (avagin) [reply CONTRIBUTING.md]: It was a joke;)

**Comment by avagin** [comment]
lgtm

File: CONTRIBUTING.md (hunk 2 above)
Comment (adrianreber) [reply CONTRIBUTING.md:1]: That sounds like a good idea. No need to have the badges in multiple locations.

**Pull request merged**